smoothchecker.py

```diff
@@ -193,49 +193,44 @@ def check_single_chunk(base_url, chunks_quality, chunk_name):
     chunk_url = base_url + '/' + chunks_quality + '/' + chunk_name
     try:
         response = _check_single_chunk(chunk_url)
-        return chunk_url, response.getcode()
-    except urllib2.HTTPError as e:
-        print "{} returned {}".format(e.url, e.code)
-        return e.url, e.code
-    except BadStatusLine:
-        print "{} returned bad status".format(chunk_url)
+        return chunk_url, response.status_code
+    except Exception:
```

Comment:
Here we are losing if there was a 404 of for example a bad status line. With verizon we had both of them.
Although most times, BadStatusLine was status_code==''